node/src/components/deploy_acceptor.rs

```diff
@@ -242,6 +242,7 @@ impl DeployAcceptor {
             &self.chain_name,
             &self.deploy_config,
             self.max_associated_keys,
+            verification_start_timestamp,
```

Comment:
You mean instead of `current_node_timestamp`?  I agree, and I'd be inclined to make that change in this PR too.

@rafal-ch - any objections if I do that?

Reply:
Done in [b25ef5b](https://github.com/casper-network/casper-node/pull/3972/commits/b25ef5b0a542ed178b234bcefdb771fba9c8efb2).